client/state/notices/middleware.js

```diff
@@ -169,6 +183,8 @@ export const handlers = {
 	[ PUBLICIZE_CONNECTION_UPDATE_FAILURE ]: onPublicizeConnectionUpdateFailure,
 	[ GUIDED_TRANSFER_HOST_DETAILS_SAVE_SUCCESS ]: dispatchSuccess( translate( 'Thanks for confirming those details!' ) ),
 	[ SITE_FRONT_PAGE_SET_FAILURE ]: dispatchError( translate( 'An error occurred while setting the homepage' ) ),
+	[ SITE_SETTINGS_SAVE_SUCCESS ]: dispatchSuccess( translate( 'Settings saved!', { id: 'site-settings-save' } ) ),
```

Comment:
I don't think you meant for the object to be an argument to `translate`? Instead I think you want it to be the second argument to `dispatchSuccess`, but currently the `dispatchSuccess` function only accepts a `message` and not `options`.